style.css

```diff
@@ -0,0 +1,228 @@
+.mobile-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+#shape {
+  position: relative;
+  width: 74px;
+  height: 74px;
+  left: 28px;
+  top: -9px;
+}
+
+#shape1 {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+#logo {
+  position: absolute;
+  width: 74px;
+  height: 74px;
+  left: 34px;
+  top: 0;
+  background: url(images/Shape.svg);
+  background-blend-mode: normal, overlay, normal;
+}
+
+#mobile-logo {
+  position: absolute;
+  width: 72px;
+  height: 74px;
+  left: 0;
+  top: 0;
+  background: url(images/Shape1.svg);
+  background-blend-mode: normal, overlay, normal;
+}
+
+#mobile-menu {
+  display: flex;
+  height: 78px;
+  align-items: center;
+  width: 20px;
+  background: url(images/ic_menu.png);
+  background-repeat: no-repeat;
+}
+
+.main-header {
+  display: none;
+}
+
+.projects-block h2 {
+  font-size: 4em;
+  font-family: Playfair Display, serif;
+  margin-right: 20px;
+}
+
+#primary-text h1 {
+  font-size: 50px;
+  font-family: Playfair Display, serif;
+  padding: 0 20px;
+  color: #1a2236;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 100%;
+}
+
+.content p {
+  font-family: Poppins, serif;
+  padding: 0 20px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  margin: 0;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.button {
+  width: 288px;
+  height: 50px;
+  font-family: Poppins, serif;
+  color: white;
+  background-color: #1a2236;
+  align-items: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-top: 20px;
+  cursor: pointer;
+}
+
+.button:hover {
+  background-color: white;
+  color: #1a2236;
+}
+
+.geometry-image2 {
```

Comment:
- [ ] Please use `background-size: cover;` if you want to cover the width of the screen. As I can see that without this property your design is not looking good and there is a `white-space` which you can see clearly in the below table which makes your design looks unprofessional.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-01 19-58-11](https://user-images.githubusercontent.com/66476329/235467448-84826358-eaeb-4404-84cb-79cacd993e28.png)

</td>
<td>

![Screenshot from 2023-05-01 19-13-21](https://user-images.githubusercontent.com/66476329/235467463-3cb44147-0445-4899-ab8d-6fd27fff8909.png)

</td>
</tr>
</table>